redaxo/src/addons/install/lib/command/list.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @package redaxo/core
+ *
+ * @internal
+ */
+class rex_command_install_list extends rex_console_command
+{
+    protected function configure()
+    {
+        $this->setDescription('Lists available packages on redaxo.org')
+            ->addOption('search', 's', InputOption::VALUE_REQUIRED, 'filter list')
```

Comment:
entweder sollten wir hier beschreiben, dass sich der search termn nur auf addon keys auswirkt, oder in mehr spalten suchen, z.b. auch nach `author` oder `name`

Reply:
könnte man auch ausweiten, weiß net was besser ist.

Reply:
nochmal drüber nachgedacht: ich denke der search term hier sollte die gleiche "suchgrundlage" haben, wie das suchfeld im backend im installer

Reply:
guter einfall, sollte auf jeden fall einheitlich sein